release.yaml

```diff
@@ -78,6 +78,10 @@ rancher-cis-benchmark-crd:
 system-upgrade-controller:
 - 100.0.2+up0.3.2
 rancher-cis-benchmark:
-- 2.0.3
+- 2.0.3-rc1
 rancher-cis-benchmark-crd:
-- 2.0.3
+- 2.0.3-rc1
```

Comment:
should be dropped? why is this in this commit?

Reply:
Someone else did it, it's in Dev-2.6 now